packages/theming/default-theme/src/components/Button/ButtonTheme.ts

```diff
@@ -0,0 +1,34 @@
+
+import type { Theme } from '@fluentui-react-native/framework';
+import { immutableMerge } from '@fluentui-react-native/immutable-merge';
+
+import { defaultButtonColorTokens } from './ButtonColorTokens';
+import { defaultButtonFontTokens } from './ButtonFontTokens';
+import { defaultButtonTokens } from './ButtonTokens';
+import { defaultCompoundButtonColorTokens } from './CompoundButtonColorTokens';
+import { defaultCompoundButtonFontTokens } from './CompoundButtonFontTokens';
+import { defaultCompoundButtonTokens } from './CompoundButtonTokens';
+import { defaultToggleButtonColorTokens } from './ToggleButtonColorTokens';
+
+export const defaultButtonTheme = (theme: Theme) =>
+({
+  components: {
```

Comment:
What's the thinking for the older version of the Button? Although it's marked as deprecated we haven;t been able to force everyone off of it yet.

Reply:
Do you think we have users outside of win32?
I had to move GDPR to the new button because the older version was relying on some layout behavior we have in win32 that isn't in the other RN platforms.  So, the older button would layout incorrectly when I switched to fabric.

So, I guess I'm suggesting forcing people to upgrade before they can move to fabric.